lib/jwt/taskrouter/util.js

```diff
@@ -65,7 +66,99 @@ function defaultEventBridgePolicies(accountSid, channelId) {
   ];
 }
 
+/**
+ * Generate TaskRouter workspace url
+ *
+ * @param {string} [workspaceSid] workspace sid or '**' for all workspaces
```

Comment:
you should be able to do that here. Is this what you are referring to? https://github.com/twilio/twilio-node/pull/246/files#diff-cce12a1e86ded2760558610cfe5ef0bcR16